app/assets/javascripts/admin/addon/templates/permalinks.hbs

```diff
@@ -6,51 +6,56 @@
 <PermalinkForm @action={{action "recordAdded"}} />
 
 <ConditionalLoadingSpinner @condition={{this.loading}}>
-  {{#if this.model.length}}
-    <div class="permalink-search">
-      <TextField @value={{this.filter}} @class="url-input" @placeholderKey="admin.permalink.form.filter" @autocorrect="off" @autocapitalize="off" />
-    </div>
-    <table class="admin-logs-table permalinks grid">
-      <thead class="heading-container">
-        <th class="col heading first url">{{i18n "admin.permalink.url"}}</th>
-        <th class="col heading destination">{{i18n "admin.permalink.destination"}}</th>
-        <th class="col heading actions"></th>
-      </thead>
-      <tbody>
-        {{#each this.model as |pl|}}
-          <tr class="admin-list-item">
-            <td class="col first url">
-              <FlatButton @title="admin.permalink.copy_to_clipboard" @icon="far-clipboard" @action={{action "copyUrl" pl}} />
-              <span id="admin-permalink-{{pl.id}}" title={{pl.url}}>{{pl.url}}</span>
-            </td>
-            <td class="col destination">
-              {{#if pl.topic_id}}
-                <a href={{pl.topic_url}}>{{pl.topic_title}}</a>
-              {{/if}}
-              {{#if pl.post_id}}
-                <a href={{pl.post_url}}>{{pl.post_topic_title}} #{{pl.post_number}}</a>
-              {{/if}}
-              {{#if pl.category_id}}
-                {{category-link pl.category}}
-              {{/if}}
-              {{#if pl.tag_id}}
-                <a href={{pl.tag_url}}>{{pl.tag_name}}</a>
-              {{/if}}
-              {{#if pl.external_url}}
-                {{#if pl.linkIsExternal}}
-                  {{d-icon "external-link-alt"}}
+  <div class="permalink-search">
+    <TextField @value={{this.filter}} @class="url-input" @placeholderKey="admin.permalink.form.filter" @autocorrect="off" @autocapitalize="off" />
+  </div>
+
+  <div class="permalink-results">
+    {{#if this.model.length}}
```

Comment:
If we don't have a `model` or `filter` present we don't render anything atm. Is that by design?

Reply:
Somewhat, yes. If there's not `model` it doesn't render the `permalink-results` table, only the permalink form. If there is no filter it displays all the results of the model in the table. We also now only show the `no_results` message only if you had been filtering something and saw no results, rather than seeing it all the time when there is no permalinks created.

Reply:
I think we should add a message when there are no permalinks along the lines of: "Use the form above to add your first permalink.". And we shouldn't include the search form either (nothing to search).